syncapi/storage/postgres/output_room_events_table.go

```diff
@@ -81,12 +81,12 @@ const selectEventsSQL = "" +
 
 const selectRecentEventsSQL = "" +
 	"SELECT id, event_json, session_id, exclude_from_sync, transaction_id FROM syncapi_output_room_events" +
-	" WHERE room_id = $1 AND id > $2 AND id <= $3" +
+	" WHERE room_id = $1 AND id > $2 AND id <= $3 AND sender != ALL($5)" +
```

Comment:
Is `ALL` right here? I would expect it to be `ANY`.

Reply:
yeah it works
https://stackoverflow.com/a/11730845/10436060
however it would also work with ANY

Reply:
Let's use `ANY`, it feels more correct.

Reply:
ALL was suggested by @babolivier 
https://github.com/matrix-org/dendrite/pull/864#discussion_r375212389

Reply:
I've done some further testing locally and it looks like `ALL` is OK, so I'll accept this.